net/zabbix3/Portfile

```diff
@@ -46,7 +46,7 @@ configure.args      --bindir=${prefix}/bin/zabbix \
                     --sbindir=${prefix}/sbin/zabbix \
                     --libexecdir=${prefix}/libexec/zabbix \
                     --datadir=${prefix}/share/zabbix \
-                    --sysconfdir=${prefix}/etc/zabbix \
+                    --sysconfdir=${prefix}/etc/zabbix3 \
```

Comment:
This change means that the bump in minor version may suddenly cause someone's config files to no longer be looked at. What should we do?